example/game/src/main/java/no/nordicsemi/android/ble/example/game/timer/TimerViewModel.kt

```diff
@@ -0,0 +1,37 @@
+package no.nordicsemi.android.ble.example.game.timer
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.transform
+import kotlinx.coroutines.launch
+import kotlin.time.Duration.Companion.seconds
+
+object Timer {
+    const val TOTAL_TIME = 20_000L
```

Comment:
This could be a setting for the user who is a server.
Also, the user could be able to select the category and difficulty level. That would make the game more fun :)